components/crud-web-apps/jupyter/frontend/src/app/types.ts

```diff
@@ -8,7 +8,7 @@ export interface JWABackendResponse extends BackendResponse {
   vendors?: string[];
 }
 
-export type ServerType = 'jupyter' | 'vscode' | 'rstudio';
+export type ServerType = 'jupyter' | 'groupOne' | 'groupTwo';
```

Comment:
Could you replace the 
1. `groupOne` value with `group-one` 
2. `groupTwo` value with `group-two`

So that they align with the `serverType` the backend values will be checking?